docs/csharp/language-reference/operators/patterns.md

```diff
@@ -47,13 +47,15 @@ You use declaration and type patterns to check if the run-time type of an expres
 
 A *declaration pattern* with type `T` matches an expression when an expression result is non-null and any of the following conditions are true:
 
-- The run-time type of an expression result is `T`.
+- The run-time type of an expression result is exactly `T`.
```

Comment:
Let's refine this because of the identity conversions for Tuple types:

```suggestion
- The run-time type of an expression result has an identity conversion to `T`.
```